pom.xml

```diff
@@ -598,13 +598,15 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
+            <version>3.0.0-M7</version>
```

Comment:
what does it do

Reply:
who does it to?

Reply:
Addresses one of these warnings (the other is in cdap)
<img width="858" alt="Screen Shot 2022-12-16 at 5 08 47 PM" src="https://user-images.githubusercontent.com/94018249/208197075-c8c383bc-48cc-49f4-a41a-e5a348bce973.png">